modules/osqp/0.6.3/MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+module(name = "osqp", version = "0.6.3")
+
+bazel_dep(name = "qdldl", version = "0.1.7")
+bazel_dep(name = "suitesparse", version = "7.6.0")
+bazel_dep(name = "cmake_configure_file")
+git_override(
```

Comment:
Override won't work transitively, can you please check in the re-required module first?

Reply:
update [here](https://github.com/bazelbuild/bazel-central-registry/pull/1452/commits/6749eb91ab2c6ed086d26ed515915cd7d6e4b3e0)